### DISCUSSION TITLE
cellRanger-atac v2.0.0 version removed annotation tracks blacklist.bed

### DISCUSSION THREAD

**Discussion by xiaowenchenjax** [stuart-lab/signac#889] [Q&A] [answered]
Hi @timoast 
cellRanger released the new version to preprocess  ATAC-seq data. and it announced that the annotation track 'blacklist.bed' has been removed, so the column "blacklist_region_fragments" in the singlecell.csv output for the corresponding tracks are all zero.
What should I do for this updates? 
Someone said the wavelet-based peak caller of cellranger 2.0.0 does a reasonable job of detecting high signal-to-noise peaks, should I filter cells based on the Ratio reads in genomic blacklist regions? if yes, how I calculated the blacklist_region_fragments?
Thank you.

**Comment by timoast** [comment, marked as answer]
The final section in [this](https://satijalab.org/signac/articles/pbmc_vignette.html) vignette shows how to compute the fraction of counts in genomic blacklist regions. For example:

```r
pbmc$blacklist_fraction <- FractionCountsInRegion(
  object = pbmc, 
  assay = 'peaks',
  regions = blacklist_hg19
)
```

**Reply by xiaowenchenjax** [reply]
Thank you for you quick reply!

**Reply by xiaowenchenjax** [reply]
Hi @timoast 
I clicked the blue word 'this' which took me to the vignette compiled at  September 21, 2021, https://satijalab.org/signac/articles/pbmc_vignette.html
i did not see the function you just mentioned. 
what did I miss?
Thank you.

**Reply by timoast** [reply]
There is a dropdown section at the end of the vignette
